scripts/nebula.service

```diff
@@ -234,6 +234,10 @@ function status_daemon {
             port=${GREEN}${port}${NC}
         else
             port=${BLINK}${RED}${port}${NC}
+            if [[$daemon_name == nebula-storaged]]; then
+                WARN "${daemon_name} after v3.0.0 will start service until ${RED}ADD HOSTS${NC}."
```

Comment:
Could just notice the user in plain English: "Add the Storage hosts to a Nebula Graph cluster"?